queue/abstract.lua

```diff
@@ -374,6 +374,28 @@ local function make_self(driver, space, tube_name, tube_type, tube_id, opts)
     return self
 end
 
+--- Release all session tasks.
+local function release_session_tasks(connection_id)
+    while true do
+        local task = box.space._queue_taken.index.pk:min{connection_id}
+        if task == nil or task[1] ~= connection_id then
+            break
+        end
+
+        local tube = box.space._queue.index.tube_id:get{task[2]}
```

Comment:
Raw idea 2:

Snippet 1:

```lua
box.cfg{}

local space = box.schema.space.create('test')
space:create_index('pk')
box.begin()
for i = 1, 1e6 do
    space:replace{i}
end
box.commit()

local start = clock.time()
for i = 1, 1e6 do
    space:get{i}
end
print('get', clock.time() - start)

space:drop()
box.snapshot()

os.exit()
```

Snippet 2
```lua
box.cfg{}

local space = box.schema.space.create('test')
space:create_index('pk')
box.begin()
for i = 1, 1e6 do
    space:replace{i}
end
box.commit()

local start = clock.time()
for i = 1, 1e6 do
    space.index[0]:count{i}
end
print('count', clock.time() - start)

space:drop()
box.snapshot()

os.exit()
```

Snippet 1:  `get	0.72974896430969`
Snippet 2: `count	0.31964302062988`

Maybe it's not critical for _queue space